integration_test/LedgerUtils.spec.ts

```diff
@@ -0,0 +1,21 @@
+import 'mocha';
+import {expect} from 'chai';
+import * as ledgerUtils from '../src/utils/LedgerUtils';
+
+describe('getTezosPublicKey()', () => {
+    it('should correctly fetch the root key', async () => {
+        const result = await ledgerUtils.getTezosPublicKey("44'/1729'/0'/0'/0");
+        console.log(result)
```

Comment:
Let's add an `expect()` condition instead of a console output for all new tests.